### PR TITLE
Create moderation-review-guidelines.md

### PR DESCRIPTION
**Pull request by GeorgiaHCA**
This PR contains potential guidelines for the commnuity to use to review the moderation report they will be sent. It's based on previous work for the focus groups I did with @JamesSCTJ 

I'd really appreciate it if you could take a look and check for:

clarity
any typos/broken links etc. 
any poor phrasing (e.g. idiomatic language)
anything missing or which needs to be added

Still to add: 

Icons/graphs (visual is important)
examples for the key 

Thanks!

### CODE REVIEW

File: Moderation/moderation-review-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
+# Moderation Report: Review Guidelines
+
+Thank you for helping design a moderation process for AutSPACEs. Your insights will help build an online space that it is more community-led, safe, and inclusive for all.
```
Comment (lottycoupat) [inline Moderation/moderation-review-guidelines.md:3]: that is more community-led *

File: Moderation/moderation-review-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+
+* Future contributors to this project who want to add to the insights from the moderation report. 
+* Autism researchers and advocates who want to make sure they are serving the needs of autistic people.
+* Parents, friends and colleagues who want to better understand how to support an autistic person they know.
+
+**Anyone can read the quotes so it is important that you are comfortable having the information in the document be publicly available on the internet.**
+
+### 7. Create a moderation process and code of conduct for the platform
+
+Your thoughts and comments will be read by autistic researchers, and they will be used to help design a moderation process for the platform and write a code of conduct based on your expertise, views, hopes, and concerns. If you would like to be more involved in moderation, please contact Lotty Coupat.
```
Comment (lottycoupat) [inline Moderation/moderation-review-guidelines.md:93]: add email [Lotty Coupat](mailto:ccoupat@turing.ac.uk)

**Review by lottycoupat — APPROVED** [review]
This looks fab!! Super clear on what needs to be done for the report. Added a few comments :)

**Comment by GeorgiaHCA** [comment]
Faab, thanks for the review @lottycoupat

**Pull request merged**